packages/venia-ui/lib/components/MiniCart/product.js

```diff
@@ -40,20 +43,17 @@ const Product = props => {
     const { image } = item;
 
     const productImage = useMemo(() => {
-        // Sizes
         const imageProps = {
             alt: productName,
-            classes: { root: classes.image },
-            sizes: `80px`
+            classes: { root: classes.image }
         };
 
         if (!hasImage) {
             imageProps.src = transparentPlaceholder;
         } else {
             imageProps.resource = image.file;
-
-            // Should be kept in sync with product.css .image max-width.
             imageProps.resourceWidth = 80;
```

Comment:
I think this is still related to the const :D You could make this:

```
const PRODUCT_IMAGE_SIZE = 80;
...
imageProps.resourceWidth = PRODUCT_IMAGE_SIZE;
imageProps.sizes = ${PRODUCT_IMAGE_SIZE}px;
```

Reply:
Moved these to `const`s. They just happen to have the same value so I didn't define them in terms of each other, but I left a comment in the code explaining it.

https://github.com/magento/pwa-studio/pull/1871/commits/0e5fd9af4eb39b3d92cb8d33643059176245d5ff.